src/scripts/main.js

```diff
@@ -1,3 +1,36 @@
 'use strict';
 
-// write code here
+const tableNames = document.querySelectorAll('thead th');
+
+tableNames.forEach((element, index) => {
+  element.addEventListener('click', () => sortTable(index));
```

Comment:
Consider adding a check to ensure that `sortTable` is only called if the index is valid and corresponds to a sortable column.